src/makeCalculator.js

```diff
@@ -4,7 +4,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
```

Comment:
You've correctly used the arithmetic operator with the assignment operator `+=` as per the checklist item 2. This is a good practice for modifying existing values.